.travis.yml

```diff
@@ -16,14 +18,25 @@ script:
 - bundle exec rake default
 - gem build sensu-plugins-disk-checks.gemspec
 - gem install sensu-plugins-disk-checks-*.gem
+before_deploy:
+- git clone https://github.com/sensu/sensu-go-bonsai-asset.git bonsai
 deploy:
-  provider: rubygems
+- provider: rubygems
   api_key:
     secure: MA9lRNkP7h07yWS10BZ6ZdlWNdgKB4UU5OyLiwM+IFB8uWMM/l5OsCThYkPPRTNCcGR6O2QaXukgKSz8TX+c3b/+ocMa2iJRnREIkH6/F0ujdmMmtVFFh8/lUdtlxdg3i7HqhjbhZgUNPbDUI5eVO72QAb4s20Idyw92kPEFBXU=
   gem: sensu-plugins-disk-checks
   on:
     tags: true
     all_branches: true
-    rvm: 2.3.0
     rvm: 2.4.1
     repo: sensu-plugins/sensu-plugins-disk-checks
+- provider: script
+  script: bonsai/ruby-runtime/travis-build-ruby-plugin-assets.sh sensu-plugins-disk-checks
+  skip_cleanup: true
+  on:
+    tags: true
+    all_branches: true
+    rvm: 2.4.1
+env:
+  global:
+    secure: mayZ2ZMGx0v/efTaupGXs79ni272J2mjoSC6wq+YPbeTw/BV1G/oPXja83Wxnyb1XOzuWHwH5xQUv91luF8KPJp56Xafwczi96LdZx+osYavcpGo+x1u7e4YDG0TSDB5f1elMAraiMHsQPR2TQqCYXQRj16zznCX5nPVAlq+FZPYWPeoNtBLajA68ou4vNVSKswE75OFEvU1DzSHtxQ/Q3biXseZ0puhb+oyV6798l4N0tbe3NHwPaVzsFjXgacg8QCqgtooR2nVAPx3918tyNJeKJHi7RN1h0Fm25t1O5hxwqv9iPtJNh4uF9P1RASnW4a3e0Vc41HenK51AvsjutTeoSudAyCMt0YInT91a1c+9vxXppmhsknCBDy0WUMttE1MLXUgOyG9+xsTuEjnP0JoYG8Y16vEIlIvwx+vo4IqP/YIesbNZ+x14v8pWF8VxklRV7DU74llh9lecffXoG/rHj9l2evOLch1JX8A59p/JdJ4RmIitlvRg90OULa2M6Q6R52In/+CLso8z/ZbRblo6Dj33daJdLupazgTqLGNaCYxfAUOfGblp2piFpiEgYsf1JWptFwwECaIyka0saW3H0ygMUjMt6uFjkcDocchbiY/dV1T9GsX0w79Z2LJkosnpVmpheN11ICMkwZR2Qdw/HBoKbX5PHJUn1rG3Cs=
```

Comment:
this encrypted value is for a token connected to whom? We should probably generate one with the `sensu-plugin` user with a limited oauth scope, which scopes did you give your token?

Reply:
yeah right now this is associated with my account.  

Public_repo scope is all this OATH personal token uses.

Reply:
we may need to create a "machine account" that is a member of the sensu-plugins org that we can then use for automation.  Github only lets you generate oath tokens for "accounts" not "orgs"  but it looks like the best practice pattern here is to create a dedicated github account not tied to a specific person, that can be used to do org  automation tasks.

That way if a person drops out of the org, you don't have to retool the tokens associated with their account.

Reply:
yes this is what the `sensu-plugin` (no `s`) user is.

Reply:
Okay I set up the repo in travisCI's webUI to use sensu-plugin token as part of repository settings..similar to how codeclimate token is configured.